Essentials/src/com/earth2me/essentials/perm/impl/SuperpermsHandler.java

```diff
@@ -66,11 +66,13 @@ public boolean tryProvider() {
 
     public String getEnabledPermsPlugin() {
         String enabledPermsPlugin = null;
-        List<String> specialCasePlugins = Arrays.asList("PermissionsEx", "GroupManager",
-                "SimplyPerms", "Privileges", "bPermissions", "zPermissions", "PermissionsBukkit",
-                "DroxPerms", "xPerms");
         for (Plugin plugin : Bukkit.getPluginManager().getPlugins()) {
-            if (specialCasePlugins.contains(plugin.getName())) {
+            final boolean match = plugin.getName().toLowerCase().contains("permissions") ||
+                    plugin.getName().toLowerCase().endsWith("perms") ||
+                    plugin.getName().equals("GroupManager") ||
```

Comment:
Might be easier to maintain if you still keep the set of special cases and use the .contains() call but keep the .endsWith(perms) and .contains(permissions) calls to simplify it.